starknet_py/tests/e2e/fixtures/constants.py

```diff
@@ -59,6 +59,7 @@ def _get_env_lambda(env_name):
 TYPED_DATA_DIR = MOCK_DIR / "typed_data"
 CONTRACTS_DIR = MOCK_DIR / "contracts"
 CONTRACTS_COMPILED_V0_DIR = MOCK_DIR / "contracts_compiled"
+CAIRO_0_ABI_DIR = MOCK_DIR / "cairo_0_abi"
```

Comment:
nit:
```suggestion
CAIRO_0_CONTRACTS_ABI_DIR = MOCK_DIR / "cairo_0_contracts_abi"
```
Also please update further occurrences.